### DISCUSSION TITLE
有没有人和我一样连github都不知道是啥单纯为了绳网注册一个github账号的

### DISCUSSION THREAD

**Discussion by Orangered0801** [share121/inter-knot#2230] [其他]
有没有人和我一样连github都不知道是啥单纯为了绳网注册一个github账号的

**Comment by Baize666666666** [comment]
我就是

**Comment by kesiwumola** [comment]
我也

**Comment by Genshin-Neuvillette** [comment]
太对了，我也

**Comment by whatcanisaylaoda** [comment]
我听过，但不了解。

**Comment by anything12121** [comment]
有Github，但是不会用

**Reply by daofeng9612** [reply]
我也是

**Comment by 00190906** [comment]
+1

**Comment by wen439** [comment]
+1

**Comment by caoling1423** [comment]
我只会用GitHub白嫖别人发的小黄油
![b3de58bb_E792169_ef37664d](https://github.com/user-attachments/assets/0227ca92-0941-4c14-a1ca-1377f70cd1bb)

**Comment by CocoonedSoul** [comment]
听过，之前想过来白嫖代码，但是不会。有了“绳网”才开始真的学GitHub到底是个啥

**Comment by 6078HITHEDAY** [comment]
# What is github?
## GitHub是一个基于Git版本控制工具的软件源代码托管服务平台
- 它为开发者提供了一个可以共同协作、管理和存储代码的平台，使得团队成员可以合作开发软件项目，进行版本控制和代码管理。
- GitHub也成为了开源社区的重要平台，许多开源项目都托管在GitHub上，使得开发者可以方便地分享和参与开源项目的开发。
- GitHub的主要功能包括代码托管、版本控制、问题跟踪、代码审查、部署和持续集成等。
## GitHub 账户具有多方面的价值，主要体现在以下几个方面：
 
1. 项目托管与版本控制：方便您管理自己的代码项目，实现版本的追踪和回滚，确保项目的稳定性和可维护性。
2. 代码分享与开源协作：如果您愿意，可以将自己的代码开源分享，与全球开发者交流合作，共同改进和完善项目。这有助于提升您的技术声誉和影响力。
3. 学习资源丰富：可以访问到大量优秀的开源项目，从中学习先进的技术和编程思想。
4. 建立个人品牌：通过积极参与开源项目、贡献有价值的代码，能够在技术社区中建立起自己的个人品牌，增加个人在技术领域的知名度和认可度。
5. 求职优势：在求职过程中，展示自己在 GitHub 上的活跃和贡献，能够向潜在雇主证明您的技术能力和实践经验。
6. 团队协作：对于团队项目，提供了便捷的协作平台，方便成员之间的代码整合、问题追踪和进度管理。
7. 代码备份：作为一种可靠的代码备份方式，防止本地数据丢失。
## 了解基本概念：git
Git 是用于与 GitHub 进行交互的版本控制系统。您可以根据您的操作系统，从 Git 官网下载并安装适合的版本。
- 分支（Branch）：用于在不同的版本上进行独立的开发工作。
- 提交（Commit）：对代码或文件的更改记录。
- 拉取请求（Pull Request）：用于将分支中的更改合并到主分支。

- 克隆仓库到本地：在 GitHub 上找到您创建的仓库，点击“Clone or download”按钮，获取仓库的 URL。然后在本地使用 Git 命令行或图形化工具（如 GitHub Desktop）将仓库克隆到您的电脑。
- 进行开发工作：在本地对文件进行修改、添加或删除等操作。
- 提交更改：使用 Git 命令（如  git add 、 git commit -m "描述您的更- 改" ）将更改暂存并提交。
- 推送更改到 GitHub：使用  git push  命令将本地的提交推送到 GitHub 上的仓库。
- 创建拉取请求（如果需要与他人合作）：如果您在一个团队中工作，创建拉取请求以便其他人审查和合并您的更改。
- 学习更多高级功能：如 Issues（用于跟踪问题和任务）、Projects（用于项目管理）等。

**Comment by FRIKY-c** [comment]
大家会科学上网而不知道github吗

**Comment by LuAn-afl** [comment]
一样不知道做啥的但我看起来像是什么学术交流的感觉

**Reply by LeoStr4nge** [reply]
是程序员的开源代码社区啦，世界各地的程序员会在这里分享自己的代码项目，与大家交流共创

**Comment by CY00412** [comment]
俺也一样

**Comment by cheese502** [comment]
早就注册过辣（喜）

**Comment by linlu0792** [comment]
我上次搞github还是为了搞ai绘图<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/01c96af07865b0d5215c9bc97ebeca73_8859049719826705182.png" alt="青雀 溜了" width="75" >

**Comment by dttyuu** [comment]
我也是

**Comment by fairy-fire** [comment]
我就是，现在还不知道怎么发作品和换头像，甚至不知道评论怎么发图

**Comment by konghaikh** [comment]
我也不能说不知道，我在GitHub多次搞过资源与链接😉
但的确没想过注册账号

**Comment by Guessme404** [comment]
感觉自己越来越高级了

**Comment by MySoulcutting** [comment]
啊？

**Comment by yzysrewrwqr** [comment]
当年异想天开想做游戏而开的GitHub号终于有用了<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by 1ctrl-cv** [comment]
我拿github pages搭博客来着（
[点此查看我的小破站](https://blog.b84f2246.top)
